src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+#first,
+#second,
+#third {
+  height: 100px;
+}
+
+#first {
+  background-color: red;
+}
+
+#second {
+  background-color: green;
+}
+
+#third {
+  background-color: blue;
+}
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)